src/datastore_mad/remotes/common/not_supported.sh

```diff
@@ -20,6 +20,8 @@ script_name=$(basename $0)
 
 source $(dirname $0)/../../scripts_common.sh
 
+STDIN=`cat -`
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Use $(...) notation instead of legacy backticks `...`.